src/environment.rs

```diff
@@ -0,0 +1,103 @@
+/// Structure to deal with environment variables
+#[derive(Clone, Debug, PartialEq, Eq)]
+pub struct Environment {
+    /// Customized environment variables
+    pub vars: Vec<(String, String)>,
+    /// Define if the structure must inherit
+    pub inherit: bool,
+}
```

Comment:
The fields should not be `pub`. The `compile` method's output is all we should ever test in integration tests.